### DISCUSSION TITLE
Got qBitrr working with Unraid. Questions about a few things (tags, deletion, etc)

### DISCUSSION THREAD

**Discussion by devanteweary** [Feramance/qBitrr#83] [Q&A] [answered]
Hey, didn't want to create issues since these are just questions.

1. I noticed it auto-creates a few tags: qbitrr-allowed_seeding, qbitrr-ignored, qbitrr-imported. Can I stop it from creating those?

2. Is there a way to keep it from removing a torrent? I'd like it to add a new torrent for stalled torrents, but also keep the stalled torrent just in case it starts working again. And if both download, Radarr/Sonarr will pick whichever is the best on its own.

3. Is there a way set it so that it ignores certain trackers? For instance, if it sees a stalled torrent from one of my private trackers, I don't want it to delete it since it's from a private tracker.

Thanks for the info and thanks for the awesome app!

**Comment by Feramance** [comment, marked as answer]
So to answer your questions:

1. Not really, as qBitrr uses the tags and to use them they need to be created.

2. The tag `qbitrr-ignored` is for manual use when you want to set a torrent as ignored by qBitrr exactly for the type of scenario you described. Just assign it to the torrent and done.

3. Not at the moment, it's a good idea though and I might get around to it at some point. In the meantime as described above, you can use the tag `qbitrr-ignored`

Happy to help and hope this answers your questions.
